volatility3/framework/symbols/windows/extensions/gui.py

```diff
@@ -120,8 +130,193 @@ def get_threads(
 
             yield thread, process_name, process_pid
 
+    def _do_get_windows(
+        self, window, max_windows
+    ) -> Generator[Tuple[interfaces.objects.ObjectInterface, str], None, None]:
+        """
+        Recusively walks and yields the adjacent and child windows
+        """
+        seen_windows = set()
+        seen_children = set()
+
+        if window.vol.offset == 0:
```

Comment:
More pythonic to do `if not window.vol.offset` but this is technically fine?  Just that `== 0` tends to trip my "not pythonic" senses  5;P

Reply:
Fixed